extracting-data.js

```diff
@@ -5,6 +5,13 @@ const employee = {
   isMale: true,
 };
 
+let isAMale = null;
+if (employee.isMale === true) {
```

Comment:
Hi instead of using an if else if statement you can use a ternary operator instead. It is a neat way with minimal syntax to cover the if else statements. You can use it like this :
eployee.isMale === true ? isAMale = true : isAMale = false
Here is a link if you would like to further investigate using ternary operators : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator